pkg/tcpip/stack/stack.go

```diff
@@ -431,6 +431,10 @@ type Stack struct {
 	// protected by tablesMu.`
 	tables IPTables
 
+	// connTrackTable maintains a list of tuples needed for connection tracking
+	// for iptables NAT rules.
```

Comment:
Maybe add that this is protected by tablesMu.

Reply:
Done.